### PR TITLE
Tutorial - Learn how to write & test a Beam Job

### LINKED ISSUE #359: Create Tutorial - Learn how to write and test a non-trivial Beam job.
## Skills covered

Codebase Navigation,
Unit Tests,
Beam Jobs at Oppia, Testing of Beam Jobs, Drafting Associated Testing Doc.

## Similar issues

[19636](https://github.com/oppia/oppia/issues/19636),
[10828](https://github.com/oppia/oppia/issues/10828),
[11616](https://github.com/oppia/oppia/issues/11616),
[12120](https://github.com/oppia/oppia/issues/12120),
[14968](https://github.com/oppia/oppia/issues/14968)

## Existing relevant pages

[oppia/wiki/Tests](https://github.com/oppia/oppia/wiki/Tests)

[oppia/wiki/Apache-Beam-Jobs](https://github.com/oppia/oppia/wiki/Apache-Beam-Jobs)

## Example

Write a validation job to check each topic model has a corresponding topic summary model.

### PR DESCRIPTION
**Pull request by Ash-2k3**
Fix https://github.com/oppia/oppia-web-developer-docs/issues/359

https://docs.google.com/document/d/1WWIvRPLdxZnmPQMaKsWD1-YL4Izw-vj8LXHcjc7DAVk/edit#heading=h.sy0grl3gktdp

### CODE REVIEW

**Comment by Ash-2k3** [comment]
Hey @U8NWXD, PTAL at this PR, thanks!

File: Tutorial:-Learn-How-to-Write-and-Test-a-Non-Trivial-Beam-Job.md
Comment (U8NWXD) [inline Tutorial:-Learn-How-to-Write-and-Test-a-Non-Trivial-Beam-Job.md:1]: Please add a table of contents. Instructions and format here: https://github.com/oppia/oppia/wiki/Wiki-style-guide#writing-style

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: Tutorial:-Learn-How-to-Write-and-Test-a-Non-Trivial-Beam-Job.md
Comment (Ash-2k3) [reply Tutorial:-Learn-How-to-Write-and-Test-a-Non-Trivial-Beam-Job.md:1]: Done, PTAL.Thanks!

**Review by U8NWXD — CHANGES_REQUESTED** [review]
Great, thanks! One last thing--can you add this page to the sidebar (in `_Sidebar.md`) in the location we planned out in the google doc? sorry I missed this before

**Comment by Ash-2k3** [comment]
> Great, thanks! One last thing--can you add this page to the sidebar (in `_Sidebar.md`) in the location we planned out in the google doc? sorry I missed this before

Done, PTAL, Thanks!

File: _Sidebar.md [hunk 1]
```diff
@@ -83,6 +83,7 @@
       * [[Testing for Accessibility|Testing-for-Accessibility]]
       * [[Release accessibility checklist|Release-accessibility-checklist]]
       * [[Apache Beam Jobs|Apache-beam-jobs]]
+        * [[Tutorial - Learn to write and test a non-trivial Beam job|Tutorial:-Learn-How-to-Write-and-Test-a-Non-Trivial-Beam-Job]]
```
Comment (Ash-2k3) [inline _Sidebar.md:86]: Reference: https://docs.google.com/document/d/1W6r4AW7WOP5MzJhoRKRS73IhXLj-KKo3_HDRRWHg2D4/edit (Sidebar Wiki Doc)

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**